test_collections/matter/sdk_tests/support/python_testing/models/python_testing_hooks_proxy.py

```diff
@@ -133,12 +134,15 @@ def stop(self, duration: int) -> None:
         self.results.put(SDKPythonTestResultStop(duration=duration))
         SDKPythonTestRunnerHooks.finished = True
 
-    def test_start(self, filename: str, name: str, count: int) -> None:
+    def test_start(
+        self, filename: str, name: str, count: int, steps: list[str]
+    ) -> None:
```

Comment:
Is this a change in a hook method? Does this mean that there'll be a change in the SDK?

Reply:
SDK PR added into description